### PR TITLE
Shared 1.2.0

### PR DESCRIPTION
**Pull request by Eytan-Ivan-Reuven**

### CODE REVIEW

File: src/utils/rpc/services/lesson.RPCservice.ts
Comment (eilamHasson) [inline src/utils/rpc/services/lesson.RPCservice.ts:1]: make sure to delete uses in backend

File: src/config.ts [hunk 1]
```diff
@@ -43,13 +43,10 @@ export const config = {
       item: ["thumbNail"],
       unit: ["image"],
       infographic: ["image"],
```
Comment (eilamHasson) [inline src/config.ts]: delete other meaningless stuff

File: src/utils/rpc/services/item.RPCservice.ts [hunk 2]
```diff
@@ -24,40 +22,12 @@ export class ItemRPCService {
...
-    });
-  }
-
   static async createItem(item: IItem): Promise<IItem> {
     return RPCClientRequest(ItemRPCService.rpcClient, "createItem", {
       item,
     });
   }
 
-  static async getAreaById(areaId: string): Promise<IArea> {
```
Comment (eilamHasson) [inline src/utils/rpc/services/item.RPCservice.ts:45]: make sure to delete uses of those functions in backend

File: src/utils/joi/joi.types.ts [hunk 3]
```diff
@@ -72,58 +64,8 @@ export const joiContentId = Joi.string().external(
...
-        }
-      );
-      if (isIntersecting) {
-        throw new PoligonIntersectionError();
-      }
-    }
-    return value;
-  });
-
-export const joiCoordinate = Joi.array()
```
Comment (eilamHasson) [inline src/utils/joi/joi.types.ts:109]: make sure to delete uses of all of those schemas

**Review by eilamHasson — CHANGES_REQUESTED** [review]

File: src/utils/joi/joi.types.ts (hunk 3 above)
Comment (Eytan-Ivan-Reuven) [reply src/utils/joi/joi.types.ts:109]: I did.
It will merge with another cr on user-service

File: src/utils/rpc/services/lesson.RPCservice.ts
Comment (Eytan-Ivan-Reuven) [reply src/utils/rpc/services/lesson.RPCservice.ts:1]: Could you please elaborate more? Anything you know of specifically?

File: src/utils/rpc/services/item.RPCservice.ts (hunk 2 above)
Comment (Eytan-Ivan-Reuven) [reply src/utils/rpc/services/item.RPCservice.ts:45]: I did. It would be merged w/ a corresponding cr in user-service

File: src/config.ts (hunk 1 above)
Comment (Eytan-Ivan-Reuven) [reply src/config.ts]: done

File: src/utils/rpc/services/lesson.RPCservice.ts
Comment (eilamHasson) [reply src/utils/rpc/services/lesson.RPCservice.ts:1]: there is a single use in user.validator canPatchChapter, which calls lessonrpcservice.getchapterbyid

**Review by eilamHasson — APPROVED** [review]

**Review by JuiceLayer — APPROVED** [review]

**Pull request merged**